test/BasketHandlerEdgeCases.test.ts

```
@@ -0,0 +1,43 @@
+  beforeEach(async function () {
+    const BasketLib = await ethers.getContractFactory("contracts/p1/mixins/BasketLib.sol:BasketLibP1");
+    basketLib = await BasketLib.deploy();
+    await basketLib.deployed();
+
+    const BasketHandler = await ethers.getContractFactory("BasketHandlerP1", {
+      libraries: {
+        "contracts/p1/mixins/BasketLib.sol:BasketLibP1": basketLib.address,
+      },
+    });
+
+    basketHandler = await BasketHandler.deploy();
```

ESLint

[error] Delete `;`

(prettier/prettier)